plugins/candela/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "candela-plugin",
     "version": "0.1.0",
     "dependencies": {
-        "candela": "^0.12.1",
+        "candela": "^0.16.0",
```

Comment:
Normally, `~` is more strict that `^`, [with tilde allowing only patch-level changes](https://www.npmjs.com/package/semver#tilde-ranges-123-12-1).

However, in pre-1.0 packages, the two are effectively equivalent, so this would be more a style convention when applied to the version specifier for `candela` (though it would apply to `datalib`, keeping it on `1.7.*`, instead of `1.*` as it is now).